### DISCUSSION TITLE
这个S905X3的主板是什么型号的盒子？What model of box is this S905X3 mainboard?

### DISCUSSION THREAD

**Discussion by wutong10wl1** [ophub/amlogic-s9xxx-armbian#2611] [Amlogic]
这个S905X3的主板是什么型号的盒子？看配置与颜色跟HK1 BOX、X96-Max之类比较像，但认真看又不像，有人知道这个是什么型号的吗？能有适合的刷机包吗？

What model of box is this S905X3 mainboard? The configuration and color, it seems like HK1 BOX, X96-Max and so on. But it doesn't look like them seriously. Anyone know what model this is? Can you have the suitable flash package? 

![微信图片_20241020084157](https://github.com/user-attachments/assets/8f75bfac-a8dc-4049-a58d-3e16c25863fd)
![微信图片_20241020084204](https://github.com/user-attachments/assets/c00045c4-46b3-43f9-a89a-2125f7958ad3)

**Comment by ophub** [comment]
用s95x3的试试，改/boot/uEnv.txt里dtb名字，保存重启测试。

**Reply by wutong10wl1** [reply]
这个板连安卓也进不了，估计之前是被刷坏了，能用HK1 BOX、X96-Max的系统包刷上去做底包吗？
